docs(common): remove deprecated templates from docs (#2655)
telerik/blazor-docs: getting-started/vs-integration/new-project-wizard.md

### Step 2: Select Project Template

> The **Create New Project** wizard provides a project template that is a blank solution based on the Microsoft template with added Telerik UI components (NuGet package reference, assets, service, and the `TelerikRootComponent`).

Reviewer: "Blank solution" can be confusing, because it duplicates the name of an existing Visual Studio template that is truly blank. Moreover, our new template won't be blank - it will include a Drawer, Grid and some text information.

```suggestion
The **Create New Project** wizard generates a project that is based on the Microsoft Blazor template. The app includes some Telerik UI components and the [required NuGet package reference, static assets, localization service, and the `TelerikRootComponent`]({%slug getting-started/what-you-need%}).
```

> > The **Admin**, **Dashboard**, **CRUD**, **Form** and **Chart** templates are now deprecated and the source code is moved to the [blazor-ui repo](https://github.com/telerik/blazor-ui/tree/master/common/legacy-project-templates) for continued access.

Reviewer: "now" can be relevant now, but not in a few months. Avoid passive voice.

```suggestion
> The **Admin**, **Dashboard**, **CRUD**, **Form** and **Chart** templates were deprecated. Their source code is available in the [`blazor-ui` repo](https://github.com/telerik/blazor-ui/tree/master/common/legacy-project-templates).
```

> > The **Admin**, **Dashboard**, **CRUD**, **Form** and **Chart** templates were deprecated. Their source code is available in the [`blazor-ui` repo](https://github.com/telerik/blazor-ui/tree/master/common/legacy-project-templates).

Reviewer 2: ```suggestion
> The **Admin**, **Dashboard**, **CRUD**, **Form**, and **Chart** templates were deprecated. Their source code is available in the [`blazor-ui` repo](https://github.com/telerik/blazor-ui/tree/master/common/legacy-project-templates).
```
